lambda-notes/src/App.css

```diff
@@ -0,0 +1,97 @@
+body {
+  background-color: #f2f1f2;
+}
+.show {
+  display: block !important;
```

Comment:
generally want to find a way to make this rule more specific rather than using !imporant